light-client-verifier/src/verifier.rs

```diff
@@ -266,6 +299,28 @@ where
     ) -> Verdict {
         ensure_verdict_success!(self.verify_validator_sets(&untrusted));
         ensure_verdict_success!(self.validate_against_trusted(&untrusted, &trusted, options, now));
+        ensure_verdict_success!(self.check_header_is_from_past(&untrusted, options, now));
+        ensure_verdict_success!(self.verify_commit_against_trusted(&untrusted, &trusted, options));
+        ensure_verdict_success!(self.verify_commit(&untrusted));
```

Comment:
Since this implementation is a subset of `verify_misbehaviour_header()`, maybe we could call `verify_misbehaviour_header()`, and then `check_header_is_from_past()`? This would make it clear that there's only one additional check between an update and a misbehaviour header.

Although then it might be confusing to call a "misbehaviour" in an "update" verification? Or maybe we do some renaming? Not sure...

Reply:
I would rather keep it as is, since it's only a few lines of code and the one check that is omitted is clearly documented in the doc comment.